README.md

```diff
@@ -128,6 +128,30 @@ if (!supportsTime) {
 
 This will add a global `TimePolyfill` function to the page.
 
+### Shadow DOM
+
+When your code is inside a component that resides in the Shadow DOM, the polyfill may not be able to find your label
+element. For this case, you can pass your label element in directly.
+
+```
+<label id="myLabel"></label>
+<input #time type="time" id="timeInput">
```

Comment:
What is with the `#time` attribute? I don't think that is valid HTML.

Reply:
It is valid. It's an HTML variable. But I'll remove it when I make the example simpler to not use Angular & TypeScript.